extensions.json

```
@@ -1283,12 +1283,6 @@
       "download": "https://github.com/rust-lang/rls-vscode/releases/download/v0.7.8/rust.vsix",
       "version": "0.7.8"
     },
```

Zenburn extension shouldn't get deleted:

```suggestion
    },
    {
      "id": "ryanolsonx.zenburn",
      "repository": "https://github.com/ryanolsonx/vscode-zenburn-theme",
      "version": "1.0.1",
      "checkout": "8d0ea180bd58cdf2a1b88250b44ee71fb80326b6"
    },
```